core/src/main/scala/io/gearpump/transport/netty/Client.scala

```diff
@@ -200,11 +200,11 @@ object Client {
     }
   }
 
-  class ClientPipelineFactory(name: String, decoder: MessageDecoder, encoder: MessageEncoder) extends ChannelPipelineFactory {
+  class ClientPipelineFactory(name: String, conf: NettyConfig) extends ChannelPipelineFactory {
```

Comment:
There is no test cover for the pipeline factory, but there is a Netty.spec which will test the Client and Server, so the pipeline factory is embedded.